### DISCUSSION TITLE
qiankun 框架中以微应用的方式可以加载webassembly项目吗

### DISCUSSION THREAD

**Discussion by yangfangfang1204** [umijs/qiankun#2981] [Q&A]
1、我们在qiankun框架中以微应用的方式可以加载了webassembly项目，但document.currentScript获取不到值，因此我们写死了document.currentScript。
2、写死document.currentScript后，加载player.wasm库时又出现了报错。
cannot read properties of undefined(reading ‘apply’) at _emscripten_receive_onmain_thread_js.
有没有相关的技术文档介绍如何加载webassembly项目，并去使用他？

**Comment by wjm0913** [comment]
解决了吗，我前几周使用了额，没有问题，资源放在了基座中，子系统去使用的。没出现啥问题。
